app/src/main/java/io/github/droidkaigi/confsched2017/model/Session.java

```diff
@@ -87,26 +86,25 @@
     private enum Type {
         CEREMONY, SESSION, BREAK;
 
-        @Override
-        public String toString() {
-            return super.toString().toLowerCase();
+        boolean matches(String type) {
+            return name().equalsIgnoreCase(type);
```

Comment:
wow 😮 interesting.
I didn't know that and take a quick search, then javadoc and some blog posts are pointing 
that `equalsIgnoreCase()` is locale *in*dependent.
https://developer.android.com/reference/java/lang/String.html#equalsIgnoreCase(java.lang.String)
http://yuki312.blogspot.jp/2013/05/androidtolowercasetouppercase.html
I'm not tested myself but if you have some clarification, plz let me know. thx!